posthog/api/test/test_survey.py

```diff
@@ -1330,6 +1335,39 @@ def test_responses_count(self):
         data = response.json()
         self.assertEqual(data, survey_counts)
 
+    def test_responses_count_only_after_first_survey_created(self):
+        survey_counts = {
+            "d63bb580-01af-4819-aae5-edcf7ef2044f": 3,
+            "fe7c4b62-8fc9-401e-b483-e4ff98fd13d5": 6,
+            "daed7689-d498-49fe-936f-e85554351b6c": 100,
+        }
+
+        expected_survey_counts = {
+            "d63bb580-01af-4819-aae5-edcf7ef2044f": 3,
+            "fe7c4b62-8fc9-401e-b483-e4ff98fd13d5": 6,
+        }
+
+        earliest_survey = Survey(team_id=self.team.id, created_at=datetime.now() - timedelta(days=101))
+        earliest_survey.save()
```

Comment:
Small nit, you can use `Survey.objects.create()` to save the survey instance directly.